pkg/workloads/cortex/client/setup.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from setuptools import setup, find_packages
+from setuptools.command.install import install
+
+import sys
+
+
+class InstallBinary(install):
+    def run(self):
+        install.run(self)
+
+        import requests
+        from pathlib import Path
+        import sys
+        import os
+        import stat
+        import shutil
+
+        dest_dir = os.path.join(self.install_lib, "cortex", "binary")
+
+        zip_file_path = os.path.join(dest_dir, "cli.zip")
+        cli_file_path = os.path.join(dest_dir, "cli")
+
+        if not os.path.exists(cli_file_path):
+            platform = sys.platform
+
+            if not platform.startswith("darwin") and not platform.startswith("linux"):
```

Comment:
if it starts with "linux", should we set it to "linux"? otherwise could line 49 fail? (same with darwin)

Reply:
Yup, good catch.